modules/nf-core/igvreports/main.nf

```diff
@@ -8,7 +8,7 @@ process IGVREPORTS {
         'biocontainers/igv-reports:1.12.0--pyh7cba7a3_0' }"
 
     input:
-    tuple val(meta), path(sites)
+    tuple val(meta), path(sites), path(tracks), path(tracks_indicies)
```

Comment:
Why the tracks_indicies if you are not using it in the module? 
Does the tool assume it to be in the same dir?

Reply:
for BAM/CRAM at least, yes